### PR TITLE
Add EventHandler to know Preloaded completed

### PR DESCRIPTION
**Pull request by KimDaeYu**
Related to https://github.com/planetarium/UniLibplanet/issues/38

Add EventHandler to know Preloaded completed. 
When an event occurs, `StartMiner` is executed.
Regardless of the argument of `_miner.CoStart()`

### CODE REVIEW

**Review by Akamig — APPROVED** [review]
LGTM

File: Libplanet.Unity/SwarmRunner.cs [hunk 1]
```diff
@@ -37,6 +37,16 @@ public class SwarmRunner
             _cancellationTokenSource = new CancellationTokenSource();
         }
 
+        /// <summary>
+        /// EventHandler checked if IsPreloaded has changed.
+        /// </summary>
+        public delegate void EventHandler();
```
Comment (Akamig) [inline Libplanet.Unity/SwarmRunner.cs:43]: As following microsofts naming convention, seems there're more option to naming this event handler.
https://docs.microsoft.com/en-us/dynamicsax-2012/developer/naming-conventions-delegates-and-event-handlers

File: Libplanet.Unity/SwarmRunner.cs [hunk 2]
```diff
@@ -37,6 +37,16 @@ public class SwarmRunner
...
+
+        /// <summary>
+        /// Event checked if IsPreloaded has changed.
+        /// </summary>
+        public event EventHandler? IsPreloadedChanged;
```
Comment (Atralupus) [inline Libplanet.Unity/SwarmRunner.cs:48]: This EventHandler itself runs only once time when the preload is finished.
I don't think there will be a situation where I can receive an event that it has been preload changed.
Therefore, it seems better to delete the IsPreloaded that exists as the current variable and change it to EventHandler IsPreloaded.
Also, there seems to be little need to be public. better to change it to private.

**Comment by KimDaeYu** [comment]
It may be necessary to change the access level of `PreloadedEventHandler` and `Preloaded` in `SwarmRunner.cs`.  
I don't know how to fix it, so I implemented it as public first.

**Review by Atralupus — APPROVED** [review]

**Comment by Atralupus** [comment]
> It may be necessary to change the access level of `PreloadedEventHandler` and `Preloaded` in `SwarmRunner.cs`.
> I don't know how to fix it, so I implemented it as public first.

OK👍 Good, Thanks for your contributing

**Pull request merged**